alembic/versions/482cd6517ce4_add_default_user.py

```
@@ -21,7 +21,10 @@
 
 
 def upgrade() -> None:
-    await_only(create_default_user())
+    try:
+        await_only(create_default_user())
+    except Exception:
+        pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)